Description: library: days/users when value is 0
 The check for using days/users was > 0 which meant you could
 get results like 0 day or 0 user. The check is now for not 1
 so w prints 0 days and 0 users.
Origin: upstream, https://gitlab.com/procps-ng/procps/-/commit/91ff8d99592afd0906cc564b46bbd503a9d4175f
Applied-Upstream: 4.0.5
Last-Update: 2025-04-14
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/library/uptime.c
+++ b/library/uptime.c
@@ -161,7 +161,8 @@
         realtime.tm_hour, realtime.tm_min, realtime.tm_sec);
 
     if (updays)
-        pos += sprintf(upbuf + pos, "%d %s, ", updays, (updays > 1) ? "days" : "day");
+        pos += sprintf(upbuf + pos, "%d %s, ", updays, (updays != 1) ? "days" : "day");
+
 
     if (uphours)
         pos += sprintf(upbuf + pos, "%2d:%02d, ", uphours, upminutes);
@@ -177,7 +178,7 @@
       pos += sprintf(upbuf + pos, "%2d ", users);
 
     pos += sprintf(upbuf + pos, "%s,  load average: %.2f, %.2f, %.2f",
-        users > 1 ? "users" : "user",
+        users != 1 ? "users" : "user",
         av1, av5, av15);
 
     return upbuf;
@@ -269,7 +270,7 @@
     if (updays) {
         pos += sprintf(shortbuf + pos, "%s%d %s",
                        comma  > 0 ? ", " : "", updays,
-                       updays > 1 ? "days" : "day");
+                       updays != 1 ? "days" : "day");
         comma += 1;
     }
 
